pkg/cvo/cvo.go

```diff
@@ -452,3 +452,8 @@ func (optr *Operator) currentVersion() configv1.Update {
 		Payload: optr.releaseImage,
 	}
 }
+
+// SetSyncWorkerForTesting updates the sync worker for whitebox testing.
+func (optr *Operator) SetSyncWorkerForTesting(worker ConfigSyncWorker) {
```

Comment:
Hmm...

Reply:
I contemplated having start lazily configure the worker, but this is a reasonable taint.